### PR TITLE
publisher web, jndi remove, javadoc, modified sql script

### PR DESCRIPTION
**Pull request by KatalinKis**
removed jndiName from interfaces in common
added javadoc
added publisher management to client layer
modified sql script- had incorrect data

### CODE REVIEW

File: wildfly-users-web/src/main/java/gallb/wildfly/users/web/PublisherManagedBean.java [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+
+/**
+ * Publisher manager. Invokes the CRUD methods of the server
+ * 
+ * @author kiska
+ *
+ */
+@Named("publisherbean")
+@SessionScoped
+public class PublisherManagedBean implements Serializable {
```
Comment (todor-tibor) [inline wildfly-users-web/src/main/java/gallb/wildfly/users/web/PublisherManagedBean.java]: A tobbi managed bean osztaly ugy lett elnevezve hogy: PublicationMB, AuthorMB stb.
Kerlek tartsuk ezt a naming convention-t: tehat PublisherMB

File: wildfly-users-web/src/main/java/gallb/wildfly/users/web/PublisherManagedBean.java [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+	 * Searches for all publishers whom name contain the {@code p_searchTxt}
+	 * input data
+	 * 
+	 * @param p_searchTxt
+	 *            - a string containing the search term
+	 * @return - a list of publisher objects if any matches were found
+	 */
+	public List<Publisher> search(String p_searchTxt) {
+		if (p_searchTxt.length() >= 3) {
+			publishersList = new ArrayList<>();
```
Comment (todor-tibor) [inline wildfly-users-web/src/main/java/gallb/wildfly/users/web/PublisherManagedBean.java]: minden keresesnel uj ArrayList objektum. Lasd az AuthorMB-t.

**Comment by KatalinKis** [comment]
kert modositasok megcsinalva

**Review by todor-tibor — APPROVED** [review]
ok

**Pull request merged**